src/components/modules/blockManager.ts

```diff
@@ -573,23 +571,23 @@ export default class BlockManager extends Module {
 
     const parentFirstLevelBlock = (childNode as HTMLElement).closest(`.${Block.CSS.wrapper}`);
 
-    if (parentFirstLevelBlock) {
-      /**
-       * Update current Block's index
-       *
-       * @type {number}
-       */
-      this.currentBlockIndex = this._blocks.nodes.indexOf(parentFirstLevelBlock as HTMLElement);
+    if (!parentFirstLevelBlock?.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)) {
```

Comment:
Support multiple Editor.js instances by checking the equality between `this.Editor.UI.CSS.editorWrapper` and `this.Editor.UI.nodes.wrapper`.